cf-agent/verify_files.c

```diff
@@ -415,7 +415,7 @@ static PromiseResult VerifyFilePromise(EvalContext *ctx, char *path, const Promi
 
 /* Phase 1 - */
 
-    if ((exists && ((a.haverename) || (a.haveperms) || (a.havechange) || (a.transformer))) ||
+    if ((exists && ((a.haverename) || (a.haveperms) || (a.havechange) || (a.transformer)) || (a.acl.acl_entries != NULL)) ||
```

Comment:
The compiler is warning here, and on a second look the new check is not on the right parenthesis level, there is a slight logic error here. 

I'm a bad reviewer it seems. :-) I'll adjust and commit.